fusor-ember-cli/app/templates/abandon-deployment-modal.hbs

```diff
@@ -0,0 +1,20 @@
+{{#em-modal configName="bs" id="abandonDeploymentModal" open-if=isAbandonModalOpen close-if=isCloseModal}}
+    {{#em-modal-title}}
+        {{#em-modal-toggler class="close"}}
+            <span aria-hidden="true">×</span><span class="sr-only">Close</span>
+        {{/em-modal-toggler}}
+        <h4 class="modal-title">Abandon QCI Deployment</h4>
```

Comment:
We're replacing RHCI with QCI (Quickstart Cloud Installer) as part of the rebranding effort on all copy. I'll double check I have that acronym right, but apparently RHCI is no more!